.github/workflows/release.yml

```
@@ -51,7 +51,7 @@ jobs:
         run: |
           echo "Checking for changes..."
           # Get the list of changed packages
-          CHANGED_PACKAGES=$(yarn lerna changed --json | jq -r '.[].name' || echo "")
+          CHANGED_PACKAGES=$(yarn lerna changed --json | sed -n '/^\[/,/^\]/p' | jq -r '.[].name' || echo "")
```

```suggestion
          CHANGED_PACKAGES=$(yarn --silent lerna changed --json | jq -r '.[].name' || echo "")
```

だとどうですかね、ローカルで試したらyarn自体がいろいろ出力していたので……

---

同じくローカルでいけました〜のでそっちでやってみましょか！